httpx/_client.py

```diff
@@ -90,7 +89,7 @@ def _get_proxy_map(
         if proxies is None:
             if trust_env:
                 return {
-                    key: Proxy(url=url)
+                    key: None if url is None else Proxy(url=url)
```

Comment:
Previously `get_environment_proxies()` did not return `None` values.
Now it does.